helpers/helpers.go

```diff
@@ -400,18 +400,24 @@ func CheckTokenExpiration(accessToken string) error {
 	switch untilExp := time.Until(expiration); {
 	case untilExp < 0:
 		return fmt.Errorf("the provided access token has expired, please renew it")
-	case untilExp > 0 && untilExp < 24*time.Hour:
-		fmt.Fprintln(
+	case untilExp >= 0 && untilExp < 2*time.Hour:
+		fmt.Fprintf(
 			os.Stderr,
-			"The provided access token expires in",
-			time.Until(expiration).Truncate(time.Second),
+			"WARNING! The provided access token expires in only %d hours and %d minutes.\n",
+			int(untilExp.Hours()), int(untilExp.Minutes())-(int(untilExp.Hours())*60),
+		)
+		fmt.Fprintln(os.Stderr, "Consider renewing the token.")
+	case untilExp >= 2*time.Hour && untilExp < 24*time.Hour:
+		fmt.Fprintf(
+			os.Stderr,
+			"The provided access token expires in %d hours and %d minutes (at %d:%d).\n",
+			int(untilExp.Hours()), int(untilExp.Minutes())-(int(untilExp.Hours())*60), expiration.Hour(), expiration.Minute(),
```

Comment:
Having the time remaining and the timestamp here feels a bit off. It's better to be consistent and only provide the remaining time.